build.sh

```diff
@@ -23,6 +23,22 @@ usage()
     exit 1
 }
 
+initDistroName()
+{
+    if [ "$__BuildOS" == "Linux" ]; then
+        # Detect Distro
+        if [ "$(cat /etc/*-release | grep -cim1 ubuntu)" -eq 1 ]; then
+            export __DistroName=ubuntu
+        elif [ "$(cat /etc/*-release | grep -cim1 centos)" -eq 1 ]; then
+            export __DistroName=centos
```

Comment:
Yes, please change __DistroName to `rhel`.  It should be fine to have both centos and rhel builds produce packages  with the 'rhel.7' RID for now.